bin/vitamin.js

```diff
@@ -6,15 +6,17 @@ import path from 'path';
 import rimraf from 'rimraf';
 import { spawn } from 'child_process';
 import fs from 'fs';
+import { spawn as npmRunSpawn } from 'npm-run';
```

Comment:
Using npm-run guarantees us that it will run jest from vitamin's node_modules. If we don't use it, it can causes issues when jest is called from another place (such as linked dependencies or global install)